static/scss/partials/main.scss

```diff
@@ -2367,12 +2368,20 @@ input.input-has-error {
   }
 }
 
+.messages {
+  width:100%;
+  padding:1.5rem 16px 0 16px;
```

Comment:
Let's not mix `rem` and `px`. 

Can you use `rem` for all of it?

Reply:
For this, should I be using protocol?